### ISSUE TITLE
support for import.meta

### ISSUE THREAD

**Issue by johndeighan** [jashkenas/coffeescript#5356]
bug report - it's not listed under **Unsupported ECMAScript Features**

### Input Code

```coffee
import { dirname } from 'path';
import { fileURLToPath } from 'url';

__dirname = dirname(fileURLToPath(import.meta.url));
```

### Expected Behavior

This should give us the equivalent to __dirname in common js

### Current Behavior

Error message for the period after **import** in the last line

### Possible Solution

Should work as it does in ECMAScript, or at least be listed as not compatible

### Context

As a workaround, I put the **import.meta.url** inside backquotes so coffeescript uses it as is.
But I'm not sure yet whether that always works.

### Environment

* CoffeeScript version: CoffeeScript version 2.5.1
* Node.js version: v16.2.0

**Comment by wdavidw** [comment]
This is the first time in 10 years that I am using the backquote behavior, thanks for the temporary solution.

**Comment by edemaine** [comment]
It looks like this bug was fixed by #5324 (June 3, 2020) but that CoffeeScript hasn't released since then (v2.5.1 was released on NPM on January 31, 2020). Probably worth a new release?

**Comment by GeoffreyBooth** [comment]
Yeah. There’s also top-level `await` that we need to support.

**Comment by GeoffreyBooth** [comment]
Resolved in #5374.
